pasqal_cloud/client.py

```diff
@@ -36,12 +36,12 @@ class Client:
 
     def __init__(
         self,
-        group_id: str,
+        project_id: Optional[str] = None,
```

Comment:
Why optional ?

Reply:
Error on my side for this one, forgot to change it, mb

Reply:
Actually i remember now why I used, it is cause of Mypy, because this project_id or group_id of the sdk are optional so the project_id has to be optional too

Reply:
It's optional in the SDK but we raise an error if not set, so mypy should not raise an error if this is strictly set to `str`. However you need to also unset the default value `= None` in the `__init__` function here

Reply:
Even if I changed it I'm still getting error: Argument "project_id" to "Client" has incompatible type "Optional[str]"; expected "str"  [arg-type]